src/layout/OrgLayout.tsx

```diff
@@ -1,19 +1,27 @@
-import getOrg from '../fetching/getOrg';
-import { useQuery } from 'react-query';
+import DefaultLayout from './DefaultLayout';
+import OrgHeader from './OrgHeader';
 
 interface OrgLayoutProps {
     children: JSX.Element;
     orgId: string;
 }
 
 const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
-    const orgQuery = useQuery(['org', orgId], getOrg(orgId));
-
     return (
-        <div>
-            <h1>{ orgQuery.data.title }</h1>
+        <DefaultLayout>
+            <OrgHeader orgId={ orgId }/>
```

Comment:
You could pass the actual organization data here instead of just passing it's ID.